ScratchDjango/User/views.py

```diff
@@ -53,7 +55,7 @@ def register_user(request):
 def login_user(request):
     user = authenticate(email=request.data["email"], password=request.data["password"])
     if user:
-        if not user.otp_enabled:
+        if user.otp_enabled=="false":
```

Comment:
I don't think this will return the right result.

Reply:
Oh yes, I was testing it with the django templates login page and it was returning false for no value selected. Forgot to change it for react. Will change it